profile-widget/profilescene.cpp

```diff
@@ -164,6 +164,7 @@ ProfileScene::ProfileScene(double dpr, bool printMode, bool isGrayscale) :
 	addTissueItems<0,16>(dpr);
 
 	percentageItem->setZValue(1.0);
+	profileYAxis->setGridIsMultipleOfThree( qPrefDisplay::three_m_based_grid() );
```

Comment:
Notice, that only profileYAxis receives a call to set grid as multiple of three.